packages/dashboard/src/components/resourceExplorer/components/panel.tsx

```diff
@@ -32,69 +41,99 @@ const PanelAssetPropertyDragHandle = ({ item }: { item: ExtendedPanelAssetSummar
   messageOverrides: DashboardMessages;
+  asset: DescribeAssetResponseWithChildAssets | null;
 }
 
 export const IotResourceExplorerPanel: React.FC<IotResourceExplorerPanelProps> = ({
-  panelItems,
   handlePanelItemClick,
+  asset,
   messageOverrides,
 }) => {
   const handlePanelItemClickInner = (e: Event, item: ExtendedPanelAssetSummary) => {
     e.preventDefault();
     handlePanelItemClick(item);
   };
 
-  const PanelCell = ({ item }: { item: ExtendedPanelAssetSummary }) => {
-    if (item?.isHeader) {
-      return <Box variant="awsui-key-label">{item.name}</Box>;
-    }
+  const Header = ({ children }: { children: React.ReactNode }) => (
+    <Box variant="awsui-key-label" display="inline">
+      {children}
+    </Box>
+  );
 
-    if (item?.isAssetProperty) {
-      return <span>{item.name}</span>;
-    }
+  const Text = ({ children }: { children: React.ReactNode }) => <Box>{children}</Box>;
 
-    return (
+  const LinkItem = ({ item }: { item: ExtendedPanelAssetSummary }) => (
+    <div>
       <Link href="#" onFollow={(e) => handlePanelItemClickInner(e, item)}>
         {item.name}
       </Link>
-    );
-  };
+    </div>
+  );
 
-  const tableColumnDefinitions = [
-    {
-      id: 'drag',
-      header: null,
-      width: '45px',
-      cell: (item: ExtendedPanelAssetSummary) => item?.isAssetProperty && <PanelAssetPropertyDragHandle item={item} />,
-    },
-    {
-      id: 'variable',
-      header: '',
-      maxWidth: '100%',
-      cell: (item: ExtendedPanelAssetSummary) => <PanelCell item={item} />,
-    },
-  ];
+  const hasChildAssets = !!asset?.children?.length;
+  const hasProperties = !!asset?.assetProperties?.length;
+  const hasAlarms = !!asset?.assetCompositeModels?.length;
+  const isEmpty = !hasChildAssets && !hasProperties && !hasAlarms;
 
   return (
-    <Table
-      variant="embedded"
-      columnDefinitions={tableColumnDefinitions}
-      items={panelItems || []}
-      trackBy="name"
-      empty={<PanelEmpty messageOverrides={messageOverrides} />}
-    />
+    <>
+      {isEmpty && <PanelEmpty messageOverrides={messageOverrides} />}
+
+      {hasChildAssets ? (
+        <details open>
+          <summary>
+            <Header>Child Assets</Header>
+          </summary>
+          {asset?.children.map((item) => (
+            <LinkItem item={item} key={item.id} />
+          ))}
+        </details>
+      ) : (
+        <></>
+      )}
+
+      {hasProperties ? (
+        <details open>
+          <summary>
+            <Header>Properties</Header>
+          </summary>
+          {asset?.assetProperties?.map((item) => (
+            <PanelAssetPropertyDragHandle item={item} key={item.id} />
+          ))}
+        </details>
+      ) : (
+        <></>
+      )}
+
+      {hasAlarms ? (
+        <details open>
+          <summary>
+            <Header>Alarms</Header>
+          </summary>
+          {asset?.assetCompositeModels?.map(({ id, name }) => (
```

Comment:
This id will not be the id of the alarm state property id. That is the `AWS/ALARM_STATE` and what we will need for the query.

Reply:
Thanks, that's helpful.